Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.opensource.zalan.do/stups/node:6.9-cd32
```

Comment:
Did we get the chance to test this big jump ? If not, imho, I think it would be safer to stick with the same version. wdyt ? 

Maybe the following snippet could help once decided to preserve the same version.
```dockerfile
FROM registry.opensource.zalan.do/library/ubuntu-18.04:latest

ENV NODE_VERSION 6.9.4

RUN apt update && \
    apt install -y --no-install-recommends curl ca-certificates && \
    curl "https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.gz" --output "node-v${NODE_VERSION}-linux-x64.tar.gz" && \
    tar xzf node-v${NODE_VERSION}-linux-x64.tar.gz -C /opt && \
    ln -s /opt/node-v${NODE_VERSION}-linux-x64/bin/node /usr/local/bin/node && \
    apt-get purge -y curl && \
    apt-get clean -y && \
    apt-get autoremove -y && \
    rm -rf /tmp/* /var/tmp/* && \
    rm -rf /var/lib/apt/lists/*

RUN node --version
```

Reply:
The service comes up and the client bundle is being built with Node 12. I couldn't test the local instance very well, because it is really tricky to get it up and running in a non-production environment.

Let me try the alternative that you have suggested and deploy it. Since we have a staging build , the portal can be tested out (atleast bundling + server request handling).